ibis/backends/tests/test_generic.py

```diff
@@ -30,6 +30,43 @@ def test_fillna_nullif(backend, con, expr, expected):
         assert con.execute(expr) == expected
 
 
+@pytest.mark.only_on_backends(['pandas', 'dask', 'pyspark'])
```

Comment:
do these work on other backends?

Reply:
They error out in SQL, impala, etc. https://github.com/ibis-project/ibis/runs/3196422295